### ISSUE TITLE
Inform/submit PRs to external parties to update Spyder website and doc links

### ISSUE THREAD

**Issue by CAM-Gerlach** [spyder-ide/spyder-docs#39]
## Issue Description

Implicitly blocked by the other I.P.R. issues; this should be done last, of course.

Some of these are even still using the old Google Code "spyderlib" link

Parties to contact:

* [x] [WinPython](https://winpython.github.io/)
    * [x] Update link on website and related changes: PR MERGED
    * [x] Update links in packages config file: DONE
    * [x] Included Programs on wiki: DONE
* [x] [Pylint](https://pylint.readthedocs.io/en/latest/user_guide/ide-integration.html): PR MERGED
    * [x] [Pycodestyle wiki](https://github.com/PyCQA/pycodestyle/wiki/RelatedTools): DONE
* [x] [SciPy](https://www.scipy.org/topical-software.html)
    * [x] Update and add links and description on website: PR MERGED
    * [x] Link in main package ``HACKING.rst.txt`` PR MERGED
* [x] [Pandas](https://pandas.pydata.org/pandas-docs/stable/ecosystem.html): PR MERGED
* [x] [Numpy](https://github.com/numpy/numpy/blob/master/doc/source/dev/development_environment.rst): PR MERGED
* [x] [IPython](https://github.com/ipython/ipython/wiki/Projects-using-IPython): DONE
* [x] NumFOCUS
    * [x] [Affiliated Project Page](https://www.numfocus.org/sponsored-projects/affiliated-projects): DONE
    * [x] [GSOC repo](https://github.com/numfocus/gsoc/pull/279): PR MERGED
* Planet feeds
    * [x] [Python Planet](https://github.com/python/planet/blob/master/config/config.ini): PR MERGED
    * [x] [Planet SciPy](http://planet.scipy.org/): DONE
* [x] [Python wiki](https://wiki.python.org/moin/IntegratedDevelopmentEnvironments) (various pages): DONE
     * [x] [Integrated Development Environments](https://wiki.python.org/moin/IntegratedDevelopmentEnvironments)
     * [x] [Python Editors](https://wiki.python.org/moin/PythonEditors)
     * [x] [Python Distributions](https://wiki.python.org/moin/PythonDistributions?highlight=%28Spyder%29)
     * [x] [PyQt - Some Existing Applications](https://wiki.python.org/moin/PyQt/SomeExistingApplications)
     * [x] [Beginner's Guide: Mathematics](https://wiki.python.org/moin/BeginnersGuide/Mathematics)
* Anaconda docs
    * [x] [Spyder page](https://docs.anaconda.com/anaconda/user-guide/tasks/integration/spyder)
    * [x] [Anaconda Navigator](https://docs.anaconda.com/anaconda/navigator/)
    * [x] [Overview](https://docs.anaconda.com/anaconda/navigator/overview)
    * [x] [Package Lists](https://docs.anaconda.com/anaconda/packages/pkg-docs) (DONE)
* [x] Wikipedia pages
    * [x] [Spyder page](https://en.wikipedia.org/wiki/Spyder_(software)): DONE
    * [x] Anaconda page: DONE
    * [x] Screenshot and logo Wikimedia commons pages: DONE
    * [x] [IDE comparison](https://en.wikipedia.org/wiki/Comparison_of_integrated_development_environments#Python): DONE
* [x] PyPI (``setup.py``): PR MERGED
* conda
    * [x] [conda-forge](https://github.com/conda-forge/spyder-feedstock): PR MERGED
    * [x] [Anaconda](https://anaconda.org/anaconda/spyder): COMPLETE
* Linux distros
    * [x] ~~[Ubuntu](https://packages.ubuntu.com/search?keywords=spyder):~~ UNRESPONSIVE
    * [x] ~~[Debian](https://packages.debian.org/search?keywords=spyder):~~ UNRESPONSIVE
    * [x] [Fedora](http://rpms.famillecollet.com/rpmphp/zoom.php?rpm=spyder): COMPLETE
    * [x] [Fedora wiki](https://fedoraproject.org/wiki/Spyder): COMPLETE
    * [x] [OpenSUSE](https://software.opensuse.org/package/spyder3): COMPLETE
    * Arch Linux
        * [x] [Main](https://www.archlinux.org/packages/community/any/spyder/): COMPLETE
        * [x] [AUR](https://aur.archlinux.org/packages/spyder3-git/): COMPLETE
    * [x] [Gentoo](https://packages.gentoo.org/packages/dev-python/spyder): COMPLETE
* Tellurium distribution
    * [x] [Tellurium Spyder plugins page](http://tellurium.analogmachine.org/plugins/writing-ide-plugins/): CONFIRMED
    * [x] Update Links, etc. in [codebase](https://github.com/sys-bio/tellurium): PR MERGED
    * [x] Wiki pages
        * [x] [Packages](https://github.com/sys-bio/tellurium/wiki/Packages): DONE
        * [x] [Required Customization](https://github.com/sys-bio/tellurium/wiki/Required-Customization): DONE
        * [x] [FAQ](https://github.com/sys-bio/tellurium/wiki/FAQ): DONE
* Other:
    * [x] [GRASS GIS](https://grass.osgeo.org/get-involved/): DONE
    

Additional places with "projects using xxx" pages where should add our link(s):

* [x] [Doctr](https://drdoctr.github.io/doctr/#projects-using-doctr): PR MERGED
* [x] [Sphinx](http://www.sphinx-doc.org/en/master/examples.html): PR MERGED
* [x] [SymPy](http://www.sympy.org/en/index.html): PR MERGED
* [x] [pycodestyle](https://github.com/pycqa/pycodestyle/wiki/RelatedTools): DONE
* [x] ~~[pygments](http://pygments.org/faq/): EMAIL SENT AND [ISSUE](https://bitbucket.org/birkenfeld/pygments-main/issues/1465/add-spyder-to-list-of-who-uses-pygments)~~ ABANDONED (non-vital)
* [x] [Cython](https://github.com/cython/cython/wiki/projects): DONE

Out of scope?
* [x] ~~[Lektor](https://www.getlektor.com/showcase/)~~ OUT OF SCOPE (in final stages of completion, but well out of scope for this issue)

**Comment by CAM-Gerlach** [comment]
I'm going to get this started now, since I'm not sure how long it will be until #63 gets done and other than being broken on narrow screens/windows its a huge improvement over our current docs, and as @ccordoba12 said we need to get our PageRank up.
